app/ui-react/syndesis/src/modules/integrations/components/editor/EditorSteps.tsx

```diff
@@ -0,0 +1,166 @@
+import { getConnectionIcon, WithConnectionHelpers } from '@syndesis/api';
+import * as H from '@syndesis/history';
+import { ConnectionOverview } from '@syndesis/models';
+import {
+  ConnectionCard,
+  ConnectionsGrid,
+  ConnectionsGridCell,
+  ConnectionSkeleton,
+} from '@syndesis/ui';
+import { WithLoader } from '@syndesis/utils';
+import * as React from 'react';
+import { Translation } from 'react-i18next';
+import { UIContext } from '../../../../app';
+import { ApiError } from '../../../../shared';
+
+export interface IEditorStepsProps {
+  error: boolean;
+  includeConnectionMenu: boolean;
+  loading: boolean;
+  connections: ConnectionOverview[];
+
+  getConnectionHref(connection: ConnectionOverview): H.LocationDescriptor;
+  getConnectionEditHref?(connection: ConnectionOverview): H.LocationDescriptor;
+}
+
+export class EditorSteps extends React.Component<IEditorStepsProps> {
+  public render() {
+    return (
+      <Translation ns={['connections', 'shared']}>
+        {t => (
+          <UIContext.Consumer>
+            {({ pushNotification }) => {
+              return (
+                <WithConnectionHelpers>
+                  {({ deleteConnection }) => {
```

Comment:
Delete connection isn't needed here at all, this page doesn't require any management functions.

Reply:
That means you can get rid of the notifications, `WithConnectionHelpers` etc